force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -192,13 +213,23 @@ public with sharing class ClassesObjectsMethods {
     public static String getDayOfWeek(DateTime dt) {
         // Define a map of day of the week as a key and day number as the value
         // Monday = 1, Tuesday = 2, etc.
+        Map<String, Integer> mapOfDays = new Map<String, Integer>();
```

Comment:
Try to declare the map inside the constructor
Map<String,Integer> daysOfTheWeek = new Map<String,Integer>{ 'Monday'    => 1, .......}